nikhita_kamath/class4/homework/nkamath_loopydoopy.rb

```diff
@@ -0,0 +1,17 @@
+my_arr = [34, 57, 82, 20345, 23]
+
+def my_max(array)
```

Comment:
@nikhitakamath There's definitely a more efficient way of doing this. Want to try again?